backend/tests/Functional/Controller/ShareAuditLoggingTest.php

```
@@ -0,0 +1,322 @@
+
+        $created = $this->assertLoggedAuditEvent(SecurityAuditLogger::SHARE_CREATED);
+        self::assertSame($owner->getId(), $created->context['actor_user_id']);
+        self::assertSame($comic->getId(), $created->context['comic_id']);
+
+        $acknowledged = $this->assertLoggedAuditEvent(SecurityAuditLogger::SHARE_SENDER_RESPONSIBILITY_ACCEPTED);
+        self::assertSame($owner->getId(), $acknowledged->context['actor_user_id']);
+        self::assertSame($comic->getId(), $acknowledged->context['comic_id']);
+        self::assertNotNull($acknowledged->context['target_id']);
+        // Server-generated, and matching the timestamp on the share, which is
+        // what would actually be produced if anybody asked for evidence.
+        self::assertNotNull($acknowledged->context['accepted_at']);
```

PHPStan (2.2.7)

[error] Call to an undefined static method App\Tests\Functional\Controller\ShareAuditLoggingTest::assertNotNull().


(staticMethod.notFound)